tests/core/focusmanager/lostfocus.js

```diff
@@ -0,0 +1,36 @@
+/* bender-tags: editor,unit */
+/* bender-ckeditor-plugins: divarea */
+
+// Assert for IE8 returns different selection comparing to other browsers.
+// Normal browser <p>foo [<strong>bar</strong>]</p>
+// IE8: <p>foo [<strong>bar]</strong></p>
+if ( CKEDITOR.env.ie && CKEDITOR.env.version <= 8 ) {
+	bender.ignore();
```

Comment:
It should be `assert.ignore()` and should be put inside specific test case.

Using it in the above way would just display info that test was ignored but test cases would run anyway, failing the test suit. The `bender.ignore()` should be used in manual tests.